client/pages/profile.jsx

```diff
@@ -0,0 +1,128 @@
+import React from "react";
+import Link from "next/link";
+import { Query } from "react-apollo";
+import gql from "graphql-tag";
+import Cookie from "js-cookie";
+
+import Layout from "../components/layouts/Layout";
+
+export const GET_USER_PROFILE = gql`
+  query profile($email: String!) {
+    profile(email: $email) {
+      email
+      firstName
+      lastName
+      location
+      workType
+      skills
+      tagline
+      statement
+      experience
+    }
+  }
+`;
+
+const Profile = () => {
+  const email = Cookie.get("email");
```

Comment:
You should expose email as React context and use it with `useContext` because accessing cookie in a component means it is dependent on the global state hence bad for testability.

Reply:
Taken care of.